tests/conftest.py

```diff
@@ -154,10 +153,8 @@ def user_service(db_session, app_config):
 
 
 @pytest.yield_fixture
-def token_service(app_config, user_service):
-    return services.UserTokenService(
-        user_service, app_config.registry.settings
-    )
+def token_service(app_config):
+    return services.TokenService("secret", "salt", 21600)
```

Comment:
This is more a question than it is a suggestion, but can the service be initialized with `kwargs`? Particularly when run across someplace randomly in the codebase that doesn't have such conveniently obvious values, those can certainly help someone understand what's happening.

Reply:
Absolutely can do this.